README.md

```diff
@@ -160,6 +160,8 @@ You can load the Privacy Manager (that UI with the toggles) any time, programati
 * `.loadGDPRPrivacyManager(withId: String, tab: SPPrivacyManagerTab = .Default)` or
 * `.loadCCPAPrivacyManager(withId: String, tab: SPPrivacyManagerTab = .Default)`
 
+> **Note**: The `String` value is the privacy manager ID for the privacy manager you wish to load on demand.
```

Comment:
@andresilveirah I agree with your opinion on it. However, I have received feedback from a client unfamiliar with our product who struggled implementing this section since they thought the ID they needed to include was the `property ID` and that we would link the PM in the backend. Feel free to reject this PR and I can raise again if the same implementation error arises